processing/src/main/java/io/druid/segment/data/RoaringBitmapSerdeFactory.java

```diff
@@ -118,7 +96,7 @@ public ImmutableBitmap fromByteBuffer(ByteBuffer buffer, int numBytes)
     @Override
     public int compare(ImmutableBitmap o1, ImmutableBitmap o2)
     {
-      return RORING_COMPARATOR.compare((WrappedImmutableRoaringBitmap) o1, (WrappedImmutableRoaringBitmap) o2);
+      throw new UnsupportedOperationException();
```

Comment:
Should this be removed? I think it's useful and might be used for index-only query processing in the future. 
If this should be removed, it's better to make ObjectStrategy to not extend Comparator.

Reply:
It was used because Bitmaps are serialized using GenericIndexedWriter, which checks if objects are sorted. It makes sense for e. g. Strings, but not bitmaps, that is pointless. So I added `setObjectsNotSorted()` method to `GenericIndexedWriter` to avoid doing pointless work when serializing things such as bitmaps.

Reply:
I see. Then, ConciseBitmapSerdeFactory has the same issue.

Reply:
Fixed